tests/flytekit/unit/common_tests/test_launch_plan.py

```diff
@@ -127,19 +128,31 @@ def test_no_additional_inputs():
     assert lp.default_inputs.parameters["required_input"].required is True
 
 
-def test_schedule():
+@_pytest.mark.parametrize(
+    "schedule,cron_expression,cron_schedule",
+    [
+        (_schedules.CronSchedule("* * ? * * *"), "* * ? * * *", None),
```

Comment:
Can we add a test here for `'0/15 * * * ? *'`

Reply:
Added in 25d4ef8e62c44b1ad5b1c049e2773fb809ac6041. Sorry I messed up commits and had to force push.